### PR TITLE
Create 2020-04-15-all-hands.md

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: _posts/meeting-notes/2020-04-15-all-hands.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+- Distributed Press
+    - uv: project meets once a week slow and stedy progress
+    - uv: MVP a grant to apply to when avaliable from web monetization program
+    - bl: https://github.com/benhylau/distributed.press/blob/master/interviews.md
+    - bl: https://github.com/benhylau/distributed.press/blob/master/values.md
+    - bl: how DP can address the issues 
+    - bl: aligned with Hypha values
+- Aether
+    - bl: finally working on this again
+    - bl: burak refocusing on COVID, build a chat async comms and thinking on market position
```
Comment (benhylau) [inline _posts/meeting-notes/2020-04-15-all-hands.md:126]:
```suggestion
    - bl: burak refocusing on Aether after supporting COVID-19 efforts, building a chat with sync comms and thinking on market position
```

File: _posts/meeting-notes/2020-04-15-all-hands.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+	- #262 shortlinks api broke, el needs to fix
+	    - yj: may scrap rebrandly completely
+	- #102 no updates
+	- #250 did some hiring things today next step create hiring process
+	    - bl: doing research on domestic and remote and need to confirm with accuntent
+- Triage NEW tasks :new:
+	- None to triage
+- Assign [UNASSIGNED][l-none] tasks :briefcase:
+	- ...
+- Any isues we should discuss?
```
Comment (benhylau) [inline _posts/meeting-notes/2020-04-15-all-hands.md:75]:
```suggestion
- Any issues we should discuss?
```

File: _posts/meeting-notes/2020-04-15-all-hands.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    - #131 talked about yesterday and deplicate some internal addresses
```
Comment (benhylau) [inline _posts/meeting-notes/2020-04-15-all-hands.md:76]:
```suggestion
    - #131 talked about yesterday and duplicate some internal addresses
```

File: _posts/meeting-notes/2020-04-15-all-hands.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+        - document email setup
+        - update inventory
+        - yj to run show and tell on email
+        - using el's infra as email provider
+        - all tasks in subtasks in the issue
+    - #6 on website
+    - #5 data retention on next website version
```
Comment (benhylau) [inline _posts/meeting-notes/2020-04-15-all-hands.md:83]: What are these numbers @ASoTNetworks ?

Comment (dcwalk) [reply _posts/meeting-notes/2020-04-15-all-hands.md:83]: website

**Pull request merged**